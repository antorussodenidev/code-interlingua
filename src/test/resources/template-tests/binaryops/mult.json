{
    "x": { "nodeType": "MultiplicativeExpression", "code": "1 * 1" },
    "y": { "nodeType": "MultiplicativeExpression", "code": "2 / 3" },
}
