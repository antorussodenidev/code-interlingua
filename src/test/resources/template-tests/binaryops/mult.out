package com.example;

public class C {
  Object f() {
    return 1 * 1 - 2 / 3;
  }
  public C() {}
}